app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  has_many :events, foreign_key: :creator_id
+  has_many :invitations, dependent: :destroy
+  has_many :events, through: :invitations, dependent: :destroy
```

Comment:
This is not a good practice: your model shouldn't have two associations named the same. 
When you have this, the latter overrides the first one.
To solve this issue, you can change the second association to `attended_events` instead of `events`.
Check the comment on the invitations model.